content/birds/victor.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Victor Allen
+sub: Product designer and developer
+bird: victor
+date: 2025-01-31
+image:
+  src: birds/victor.jpg
+  position: top
```

Comment:
I think `center` might be better here? Unless you like effect of barely peeking out over content.